profile/src/v100/factors/hd_transaction_signing_factor_instance.rs

```diff
@@ -0,0 +1,202 @@
+use crate::prelude::*;
+
+/// A specialized Hierarchical Deterministic FactorInstance used for transaction signing
+/// and creation of virtual Accounts and Identities (Personas).
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub struct HDFactorInstanceTransactionSigning<E: IsEntityPath + Clone> {
+    pub factor_source_id: FactorSourceIDFromHash,
+    pub public_key: PublicKey,
+    pub path: E,
+}
+
+impl<T: IsEntityPath + Clone> HDFactorInstanceTransactionSigning<T> {
```

Comment:
Possible to `impl<T> TryFrom< HierarchicalDeterministicFactorInstance > for HDFactorInstanceTransactionSigning<T>`? probably, and more idiomatic rust I guess.